ocaml/typing/typeopt.ml

```diff
@@ -451,6 +451,9 @@ and value_kind_variant env ~loc ~visited ~depth ~num_nodes_visited
       (cstrs : Types.constructor_declaration list) rep =
   match rep with
   | Variant_extensible -> assert false
+  | Variant_with_null ->
+    (* CR layouts v3.0: change [Pgenval] here. *)
```

Comment:
Yes. Should this happen in this patch?

Reply:
It should happen when we decide on the correct representation of `value_or_null` in the middle-end.